dev-docs/decision-records/cve.md

```diff
@@ -0,0 +1,41 @@
+# Title [ADR-5]
```

Comment:
Can we have a better title ? :p

Reply:
Yep, CVE is a little too terse even for my tastes. How about:

CVE assessment policy

or something like that?